packages/daemons/queue_parser_blocks.go

```diff
@@ -90,5 +90,5 @@ func QueueParserBlocks(ctx context.Context, d *daemon) error {
 
 	host := getHostPort(nodeHost)
 	// update our chain till maxBlockID from the host
-	return UpdateChain(ctx, d, host, blockID, "rollback_blocks_1")
+	return UpdateChain(ctx, d, host, blockID)
```

Comment:
why "rollback_block_1" was deleted?

Reply:
because there is not other options. There were rollback_block_1 or rollback_block_2. If we removed rollback_block_2 then this parameter is unneccessary.